wandb/artifacts.py

```diff
@@ -64,7 +64,7 @@ class Artifact(object):
     LocalArtifactManifestEntry = collections.namedtuple('LocalArtifactManifestEntry', (
         'path', 'hash', 'local_path'))
 
-    def __init__(self, type, description=None, metadata=None):
+    def __init__(self, type, name, description=None, metadata=None, aliases=['latest']):
```

Comment:
If someone calls `artifact.aliases.append()` the mutable default can really f things up.

Reply:
Yeah. I went back to aliases being an argument to log / use artifact, so it's ok.